x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -183,7 +212,13 @@ function IntervalDocCount({
         <EuiText
           css={{ fontStyle: 'italic', fontSize: '80%', display: 'inline' }}
         >
-          ({asBigNumber(interval.eventDocCount)} events)
+          {i18n.translate('xpack.apm.intervalDocCount.TextLabel', {
+            defaultMessage: '(',
+          })}
+          {asBigNumber(interval.eventDocCount)}{' '}
+          {i18n.translate('xpack.apm.intervalDocCount.eventsTextLabel', {
+            defaultMessage: 'events)',
+          })}
```

Comment:
I think most of these were automatically generated?!

```suggestion
          {i18n.translate('xpack.apm.intervalDocCount.eventDocCount', {
            defaultMessage: '({value} events)',// You can check for plural here too to display `s` or not.
            values: {
               value: asBigNumber(interval.eventDocCount)
            }
          })}
```